lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py

```diff
@@ -250,7 +250,7 @@ def fileOptionsTabPage(tabLayout):
     cmds.setParent(topForm)
     cmds.frameLayout(label=getMayaUsdLibString("kMayaRefDescription"))
     cmds.columnLayout(adjustableColumn=True)
-    cmds.text(align="left", label=getMayaUsdLibString("kMayaRefCacheToUSDDescription"))
+    cmds.text(align="left", wordWrap=True, label=getMayaUsdLibString("kMayaRefCacheToUSDDescription"))
```

Comment:
This column is still adjustable - does it work correctly here?

Reply:
I'll re-verify this panel.

Reply:
In this dialog, the vertical space effect is less severe and appear natural. I would leave as it is now.